doc/verify/windows-10-1809/main.tf

```diff
@@ -0,0 +1,504 @@
+terraform {
+  required_version = ">= 1.1.3"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.10"
+    }
+
+    local = {
+      source  = "hashicorp/local"
+      version = "~>1.4"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "local" {
+}
+
+resource "azurerm_resource_group" "firefoxverify" {
+  name     = "FirefoxVerifyResourceGroup-${var.namespace}"
+  location = "Japan East"
+}
+
+resource "azurerm_virtual_network" "firefoxverify" {
+  name                = "firefoxverify-${var.namespace}-network"
+  resource_group_name = azurerm_resource_group.firefoxverify.name
+  location            = azurerm_resource_group.firefoxverify.location
+  address_space       = ["10.5.0.0/16"]
+}
+
+resource "azurerm_subnet" "internal" {
+  name                 = "internal"
+  resource_group_name  = azurerm_resource_group.firefoxverify.name
+  virtual_network_name = azurerm_virtual_network.firefoxverify.name
+  address_prefixes     = ["10.5.2.0/24"]
+}
+
+resource "azurerm_network_interface" "firefoxverify" {
+  name                = "firefoxverify-${var.namespace}-instance-nic"
+  location            = azurerm_resource_group.firefoxverify.location
+  resource_group_name = azurerm_resource_group.firefoxverify.name
+
+  ip_configuration {
+    name                          = "firefoxverify-${var.namespace}-instance-nic"
+    subnet_id                     = azurerm_subnet.internal.id
+    private_ip_address_allocation = "Static"
+    private_ip_address            = "10.5.2.10"
+    public_ip_address_id          = azurerm_public_ip.firefoxverify.id
+  }
+}
+
+resource "azurerm_virtual_machine" "firefoxverify_vm" {
+  name                             = "firefoxverify-${var.namespace}-vm"
+  location                         = azurerm_resource_group.firefoxverify.location
+  resource_group_name              = azurerm_resource_group.firefoxverify.name
+  network_interface_ids            = [azurerm_network_interface.firefoxverify.id]
+  vm_size                          = "Standard_B2S"
+  delete_os_disk_on_termination    = true
+  delete_data_disks_on_termination = true
+
+  storage_image_reference {
+    # Windows 10 Professional 1809
+    publisher = "MicrosoftWindowsDesktop"
+    offer     = "Windows-10"
+    sku       = "rs5-enterprise"
```

Comment:
https://learn.microsoft.com/ja-jp/azure/virtual-machines/generation-2

> 第 1 世代 VM と比較して、第 2 世代 VM ではブート時間やインストール時間が改善されている可能性があります。

How about using rs5-enterprise-g2?

Reply:
Thanks, I've updated the SKU to g2.